submissions/tavor118sn/html-css-popup/style.css

```diff
@@ -0,0 +1,158 @@
+/*
+ *
+ * Reset styles
+ *
+ */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: inherit;
+}
+
+/*
+ *
+ * General styles
+ *
+ */
+
+body {
+  font-family: Arial, serif;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  border: 0;
+  clip: rect(0 0 0 0)
+}
+
+/*
+ *
+ * Main navbar
+ *
+ */
+
+header {
```

Comment:
Avoid styling by HTML tags. [Read more](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css).